### PR TITLE
build/ops: rpm: package COPYING, move sample ceph.conf to ceph-common

### PR DESCRIPTION
**Pull request by smithfarm**

### CODE REVIEW

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-copying/

**Comment by smithfarm** [comment]
@tserong Can you review?

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-copying/b1d94a3510387cda2f12149eccc1624987710b99/

**Comment by tserong** [comment]
LGTM

File: debian/ceph-base.install [hunk 1]
```diff
@@ -11,6 +11,7 @@ usr/lib/ceph/erasure-code/*
 usr/lib/rados-classes/*
 usr/share/doc/ceph/sample.ceph.conf
 usr/share/doc/ceph/sample.fetch_config
+usr/share/doc/ceph/COPYING
```
Comment (tchaikov) [inline debian/ceph-base.install]: as discussed over IRC, i don't think we need two copies of copyright / COPYING in debian packaging.

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-copying/23a24472cfa2e346667d19a72e0472fe4b703012

Changelog:

* make PR rpm-specific; do not touch CMakeLists.txt
* move all three files in `%{_docdir}/ceph` from ceph-base to ceph-common

**Comment by smithfarm** [comment]
@liewegas Is `src/fetch_config` still needed?

**Comment by liewegas** [comment]
i don't think so

**Comment by tchaikov** [comment]
retest this please.

**Comment by smithfarm** [comment]
ChangeLog:

* add commit to drop `fetch_config`
* conflict resolution

File: ceph.spec.in [hunk 2]
```diff
@@ -883,7 +883,7 @@ install -m 0644 -D systemd/50-ceph.preset %{buildroot}%{_libexecdir}/systemd/sys
 mkdir -p %{buildroot}%{_sbindir}
 install -m 0644 -D src/logrotate.conf %{buildroot}%{_sysconfdir}/logrotate.d/ceph
 chmod 0644 %{buildroot}%{_docdir}/ceph/sample.ceph.conf
-chmod 0644 %{buildroot}%{_docdir}/ceph/sample.fetch_config
+install -m 0644 -D COPYING %{buildroot}%{_docdir}/ceph/COPYING
```
Comment (tchaikov) [inline ceph.spec.in]: @smithfarm could you remove the references to `sample.fetch_config` in `alpine/APKBUILD.in`, so we don't break the alpine build if it is not broken already @dx9 , what do you think?

**Comment by smithfarm** [comment]
ChangeLog:

* removed the references to `sample.fetch_config` in `alpine/APKBUILD.in` also

File: ceph.spec.in (hunk 2 above)
Comment (smithfarm) [inline ceph.spec.in]: @tchaikov Done!

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-06-15_17:39:27-rados-wip-kefu-testing---basic-smithi/ all issues are known ones.

**Comment by smithfarm** [comment]
@tchaikov Thanks

**Pull request merged**